tests/e2e/pageObjects/workbench-page.ts

```diff
@@ -199,13 +219,19 @@ export class WorkbenchPage {
         await t.click(this.tableViewTypeOption);
     }
 
+    //Select view option in Workbench results
+    async selectViewTypeGraph(): Promise<void> {
+        await t.click(this.selectViewType);
+        await t.click(this.graphViewTypeOption);
+    }
+
     /**
      * Send a command in Workbench
      * @param command The command
      * @param speed The speed in seconds. Default is 1
      * @param paste
      */
-    async sendCommandInWorkbench(command: string, speed = 1, paste = false): Promise<void> {
+    async sendCommandInWorkbench(command: string, speed = 1, paste = true): Promise<void> {
```

Comment:
I don't think this should be a parameter (also the speed). IMO you can do it built the function.
Do you have tests that use different speed and paste configurations?

Reply:
yes, in some tests are set different values for this parameters

Reply:
Ok, at least add the description above. Also, since you changed it from false to true, don't you update the other tests?

Reply:
yes, thanks, removed the parameters, for the rest of the tests, I did not change the paste parameter to false, because this will slow down the test